include/FeedingFanOut.h

```diff
@@ -96,7 +99,7 @@ namespace ChimeraTK {
     // Add the consuming accessors
     // TODO FanOut constructors and addSlave should get refactoring
     for(auto el : consumerImplementationPairs) {
-      addSlave(el.first, el.second);
+      addSlaveImpl(el.first, el.second);
```

Comment:
Wouldn't it be easier yet still sufficient to call it explicitly by ```FeedingFanOut<UserType>::addSlave```?

Reply:
seems it is, yes